src/theme/custom/cosmoDark/palette.js

```diff
@@ -64,6 +66,7 @@ export default {
     paper: themeColors.background,
     secondary: themeColors.backgroundVariant,
     card: themeColors.surface,
+    dialogPaper: themeColors.surface,
```

Comment:
Once again, is there any way to fusion card an dialogPaper as a unique surface type ?

Reply:
Sharing the same color does not mean the components are the same, so I don't get what you want here.